go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/grafana/terraform-provider-grafana
 
 go 1.18
 
+replace github.com/grafana/grafana-api-golang-client => github.com/grafana/grafana-api-golang-client v0.14.1-0.20221205173354-b400961adebd
```

Comment:
Considering https://github.com/grafana/grafana-api-golang-client/pull/126 was approved, will you remove this once that one is merged?